src/components/menu/menu.spec.ts

```diff
@@ -320,6 +346,85 @@ describe('MenuComponent', () => {
         expect(component.trigger.menu._isFocused$.value).toBeFalsy();
         expect(component.subMenuTrigger.menu._isHovering$.value).toBeFalsy();
         expect(component.subMenuTrigger.menu._isFocused$.value).toBeFalsy();
+    });
+});
+
+@Component({
+    selector: 'app-menu-test',
+    template: `
+    <div class="btn-group" *ngIf="showTrigger">
+        <button
+            type="button"
+            class="btn button-secondary dropdown-toggle"
+            [uxMenuTriggerFor]="menu">
+            Actions <ux-icon name="down" class="dropdown-icon-inline"></ux-icon>
```

Comment:
If you remove the `ux-icon` here then we can get rid of the IconModule dependency, making the test cleaner.